.teamcity/settings.kts

```diff
@@ -834,9 +834,10 @@ object CheckCodeStyleBranch : BuildType({
 	}
 })
 
-object RunCanaryE2eTests : BuildType({
-	name = "Canary e2e tests"
-	description = "Run canary e2e tests"
+object RunCalypsoE2eDesktopTests : BuildType({
+	uuid = "52f38738-92b2-43cb-b7fb-19fce03cb67c"
+	name = "Run Calypso e2e tests (desktop)"
+	description = "Run Calypso e2e tests (desktop)"
```

Comment:
could we add `desktop browser` here to differentiate from the "desktop app"?

I've been confusing myself with several test suite names now that the desktop app tests run in this repo too

Reply:
Good point, I'll change the names here and for the Desktop app